llama.cpp

```diff
@@ -5609,20 +5613,35 @@ static int llama_decode_internal(
     {
         auto & logits_out = lctx.logits;
 
+#ifndef NDEBUG
+        auto & logits_valid = lctx.logits_valid;
+        logits_valid.clear();
+        logits_valid.resize(n_vocab);
```

Comment:
Should this be this then?

```suggestion
        logits_valid.resize(n_tokens);
```

Reply:
Yep, thanks.